### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by ivan4th**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34767)

### CODE REVIEW

File: dind [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 885a308d6334e91e28189e31e7fc72e5bed7063c
```
Comment (bgrant0607) [inline dind]: Please undo this change.

Comment (ivan4th) [inline dind]: Oops, sorry about that. Removed.

**Comment by ymqytw** [comment]
@ivan4th Please don't touch `staging/`, because this directory is auto-generated. Your change will be overwritten in the future. You may try to fix the typo in the upstream repo, which is `emicklei/go-restful`

**Comment by ivan4th** [comment]
@ymqytw sorry, I'll be more careful next time. Included unneeded changes in this PR accidentally. Fixed.

**Comment by ymqytw** [comment]
LGTM Thanks.
@bgrant0607 Please tag for merging.

**Comment by bgrant0607** [comment]
@ymqytw try `/lgtm`
